.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+  - "3.5"
+
+install:
+  - pip install bubuku
+
+script:
+  - py.test tests
```

Comment:
Can you please explain what is in there?
I thought that for testing one need to do something like this: 
```
python3 setup.py test
```